numpy/lib/format.pyi

```diff
@@ -20,3 +20,6 @@ def read_array_header_2_0(fp): ...
 def write_array(fp, array, version=..., allow_pickle=..., pickle_kwargs=...): ...
 def read_array(fp, allow_pickle=..., pickle_kwargs=...): ...
 def open_memmap(filename, mode=..., dtype=..., shape=..., fortran_order=..., version=...): ...
+
+class NumpyUnpickler():
```

Comment:
I'm not too familiar with these stub files, but I'd expect this to just subclass from `pickle.Unpickler`.

Reply:
That's right - I updated it.